frontend/src/components/LoginForm/LoginForm.js

```diff
@@ -1,5 +1,159 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { 
+  Button,
+  CardContent,
+  Checkbox,
+  IconButton,
+  InputAdornment,
+  Grid,
+  FormControlLabel,
+  Link,
+  Paper,
+  TextField,
+  Typography
+} from '@material-ui/core';
+import visibilityIcon from '../../assets/generic/visibility.svg';
+import visibilityOffIcon from '../../assets/generic/visibility-off.svg';
+import productIcon from "../../assets/generic/product-icon.svg"
+import { emailRegex, passwordRegex } from "../../constants/regex"
```

Comment:
It's a matter of preference, I won't contest it but I don't think it's necessary to check password and email address fields correctness on Login. Or, check it, but fail silently

Reply:
@kibartas I think it makes sense to check regex here, so backend won't be spammed with incorrect login requests. It saves a tiny bit of bandwith